### DISCUSSION TITLE
Question: Running in production with sqlite

### DISCUSSION THREAD

**Discussion by jmichalek132** [healthchecks/healthchecks#528] [Q&A]
Hi, I wanted to ask if it's okay / safe to run in production with sqlite instead of postgres or mysql for simplicity. We would be okay in this case to run it in single replica.

**Comment by cuu508** [comment]
SQLite is one of the database backends that Django [officially supports](https://docs.djangoproject.com/en/3.2/ref/databases/). Healthchecks test suite [passes with SQLite](https://github.com/healthchecks/healthchecks/runs/2734011855). So it "nominally works". I have not used Healthchecks with SQLite in production, and don't know if there would be any issues.
